gubernator/view_base.py

```diff
@@ -91,7 +91,9 @@ def render(self, template, context):
         template = JINJA_ENVIRONMENT.get_template(template)
         self.response.write(template.render(context))
 
-    def check_bucket(self, prefix):
+    def check_bucket(self, prefix, local=None):
+        if local:
+            BUCKET_WHITELIST.add(prefix)
```

Comment:
Do the ALL_CAPS of the `BUCKET_WHITELIST` indicate that it's a constant? If so, either make a copy or change its case.